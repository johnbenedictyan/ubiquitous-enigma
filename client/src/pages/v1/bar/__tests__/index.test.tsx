import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { V1BarChartPage } from '..';

const renderer = createRenderer();

describe('V1 Bar Chart Page', () => {
    it('should render and match the snapshot', () => {
        renderer.render(<V1BarChartPage />);
        const renderedOutput = renderer.getRenderOutput();
        expect(renderedOutput).toMatchSnapshot();
    });
});
