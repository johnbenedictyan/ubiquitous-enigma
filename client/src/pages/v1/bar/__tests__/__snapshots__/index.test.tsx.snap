// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`V1 Bar Chart Page should render and match the snapshot 1`] = `
<div>
  <p>
    V1 Bar Chart Page
  </p>
  <CustomBarChart
    data={Array []}
  />
</div>
`;
